cmd/setup-etcd-environment/run.go

```diff
@@ -146,10 +146,35 @@ func runRunCmd(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	parsedIP := net.ParseIP(ip)
+	if parsedIP == nil {
+		return fmt.Errorf("Failed to parse IP '%s'", ip)
+	}
```

Comment:
i think we should move this to https://github.com/openshift/machine-config-operator/pull/1211/files#diff-f26a1d5dc0cdd6c1f8863a9a3d7a3630R158 so that we are erroring out only when we need to... ?

Reply:
Is there a case where we'd want to ignore a failure to parse the IP?  I know it wasn't checked before, but if this is a bad IP, the cluster is going to break badly anyway, right?